### PR TITLE
Add verbose logs mention to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Akirathan**
### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [x] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -107,6 +107,25 @@ reproduce it, the faster we can fix the bug! It's also helpful to have the
...
+the `project-manager`, or language server separately, then pass
+`--log-level trace` option. With verbose logging, there are a lot of messages
+printed to the standard output, and it is possible that on slower terminal
+emulators this will clog the terminal and hence the whole backend. To avoid
+this, we recommend redirecting the output to `/dev/null`, via a command like
+`enso -debug.verbose > /dev/null 2>&1`.
+
+The logs are kept in a central location - on Linux, they are in
+`$XDG_DATA_HOME/enso/log` ( usually `~/.local/share/enso/log`), and on Windows
+they are in `%APPDATA%\enso\log`. The log level name consists of the timestamp
```
Comment (4e6) [inline docs/CONTRIBUTING.md]:
```suggestion
The logs are kept in a central location `$ENSO_DATA_DIRECTORY/log` - on Linux,
they are in `$XDG_DATA_HOME/enso/log` (usually `~/.local/share/enso/log`), and
on Windows they are in `%APPDATA%\enso\log`, see
[distribution.md](distribution/distribution.md) for details. The log level name
consists of the timestamp
```

Comment (4e6) [reply docs/CONTRIBUTING.md]: Technically the location is described as `ENSO_DATA_DIRECTORY/log` https://github.com/enso-org/enso/blob/develop/docs/distribution/distribution.md

**Review by 4e6 — APPROVED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (4e6) [reply docs/CONTRIBUTING.md]: Also, I'm not sure if the suggestion above is properly formatted

Comment (Akirathan) [reply docs/CONTRIBUTING.md]: I have run `./run fmt`, so it should be OK now.

**Pull request merged**